ctwrap/simulation.py

```diff
@@ -202,6 +210,12 @@ def from_dict(cls, content, name=None, path=None, **kwargs):
     def _parse_output(dct, fname=None, fpath=None):
         """Parse output dictionary (hidden function)
         Overrides defaults with keyword arguments.
+        Args:
```

Comment:
Same as above - formatting of docstrings should be consistent throughout (i.e. PEP 484 etc.).